test/unit/host_test.rb

```diff
@@ -1954,6 +1954,7 @@ def to_managed!
   end
 
   test "#info ENC YAML omits root_pw when password_hash is set to Base64" do
+    skip 'Now after save, the password is ecrypted, so the host.info never shows the unencrypted password'
```

Comment:
I wasn't sure that the change in behavior is a valid one, so I wanted another pair of eyes on it before I remove the test and confirm the behavior change.
And again, as in my previous comment, in rails 4 the `after_save` hook should be removed. So I am not sure that the test will remain invalid then.